test/e2e/auction/index.test.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-undef */
+describe('Fledge', () => {
+	describe('runAdAuction', () => {
+		beforeEach(async () => {
+			await page.goto('http://localhost:3000/test/e2e/');
+		});
+
+		it('should error when no parameters sent', async () => {
+			const fledge = await page.evaluate(() => window.fledge);
+			expect(() => fledge.joinAdInterestGroup()).toThrow();
+		});
```

Comment:
should this be testing the auction stuff and not the `joinAdInterestGroup` functions